core/utils.js

```diff
@@ -518,13 +527,55 @@ Blockly.utils.tokenizeInterpolation = function(message) {
         i--;  // Parse this char again.
         state = 0;
       }
+    } else if (state == 3) {
+      if ((c == '_') || ('0' <= c && c <= '9') || ('a' <= c && c <= 'z') ||
+          ('A' <= c && c <= 'Z')) {
+        buffer.push(c);
+      } else {
+        var key = buffer.join('');
+        var keyUpper = key.toUpperCase();
+        var bklyKey = keyUpper.startsWith('BKY_') ? keyUpper.substring(4) : null;
```

Comment:
Fixed startsWith(..).

BKY is used to distinguish Blockly's string tables, from the application's string table. As we support string tables in the dev tools, I expect more developers will take advantage of this functionality, and we want to encourage forward compatibility as both string tables may grow. In Android and iOS, we will ship `bky_messages.json` and `bky_synonym.json` files, copied from the master `msgs/json/*.json` files.
My preference was `BKLY`, but iOS had already set a precident.